src/main/java/applemazer/Applemazer.java

```diff
@@ -0,0 +1,83 @@
+package applemazer;
+
+import java.util.Scanner;
+
+import commands.Command;
+
+/**
+ * Main class that runs the Applemazer chatbot.
+ */
+public class Applemazer {
+    protected static TaskList tasks;
+    protected static Scanner sc = new java.util.Scanner(System.in); // For text-based UI.
+    private static Storage storage;
+    private final Parser parser; // For text-based UI.
+    private final Ui ui;
+    private boolean isProcessing = true;
```

Comment:
Good job on being consistent with the naming standard for booleans, noticed these elsewhere as well!